test/server/headers-option.test.js

```diff
@@ -55,6 +55,32 @@ describe('headers option', () => {
     });
   });
 
+  describe('as a function', () => {
+    beforeAll((done) => {
+      server = testServer.start(
+        config,
+        {
+          headers: () => {return { 'X-Bar': ['key1=value1', 'key2=value2'] }},
+          port,
+        },
+        done
+      );
+      req = request(server.app);
+    });
+
+    afterAll(testServer.close);
+
+    it('GET request with headers as a function', (done) => {
+      // https://github.com/webpack/webpack-dev-server/pull/1650#discussion_r254217027
+      const expected = ['v7', 'v8', 'v9'].includes(
+        process.version.split('.')[0]
```

Comment:
We don't support these Node.js versions https://nodejs.org/api/process.html#process_process_version :smile:

Reply:
done